parts-kit.html

```diff
@@ -148,6 +148,36 @@ <h3 class="mb-0">Secondary</h3>
     %}
   </div>
 
+  <h3 class="mb-0">Tertiary</h3>
+
+  <div class="flex gap">
+    {% include components/button.html
+      text = "Tertiary"
+      href = "#"
+      style = "tertiary"
+    %}
+
+    {% include components/button.html
+      text = "Tertiary with Icon"
+      href = "#"
+      style = "tertiary"
+      icon = "files"
+    %}
+
+    {% include components/button.html
+      text = "Tertiary as Button"
+      tag = "button"
+      style = "tertiary"
+    %}
+
+    {% include components/button.html
+      text = "Tertiary Disabled"
+      href = "#"
+      style = "tertiary"
+      disabled = true
+    %}
+  </div>
```

Comment:
We added a new (non-official) tertiary variant to the buttons for Resource links in the Abstract section